tests/python/frontend/tflite/test_forward.py

```diff
@@ -1281,70 +1287,72 @@ def test_all_unary_elemwise():
 # Element-wise
 # ------------
 
-def _test_elemwise(math_op, data, fused_activation_function=None, quantized=False, qnn_op=None):
+def _test_elemwise(math_op, data, fused_activation_function=None, quantized=False, qnn_op=None, same_qnn_params=False):
     """ One iteration of elemwise """
 
     assert len(data) == 2
 
     # Test with two tensors
-    with tf.Graph().as_default():
-        in_data = [array_ops.placeholder(shape=data[0].shape, dtype='float32', name='in_0'),
-                   array_ops.placeholder(shape=data[1].shape, dtype='float32', name='in_1')]
-
-        if quantized:
+    def __test_elemwise( in_data ):
```

Comment:
My suggestion would be to use the TFLite 2.1 way of creating quantization tests. I think its cleaner.

An example for Relu is here
https://github.com/apache/incubator-tvm/blob/c9c77c6b76f7cff3bc6afbf9d3ef2200e3fdbb91/tests/python/frontend/tflite/test_forward.py#L2072-L2089

Reply:
I slightly simplified unit test (using current implementation). Is there an example with different quantisation parameters for two input tensors and one output tensor written using Keras?